client/my-sites/site-settings/form-seo.jsx

```diff
@@ -400,15 +404,58 @@ export const SeoForm = React.createClass( {
 					) }
 				</Card>
 
+				{ ! showAdvancedSeo &&
+					<Card className="seo-form-nudge">
+						<div className="seo-form-nudge__business-plan-card">
+							<PlanCompareCard
+								title={ getPlan( PLAN_BUSINESS ).getTitle() }
```

Comment:
Could you please populate this from the plans endpoint? This is hardcoded and will be incorrect. 

The PlanCompareCard was originally only used on the feature pages that we created during the meetup.